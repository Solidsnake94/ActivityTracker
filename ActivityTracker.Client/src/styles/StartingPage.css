form,h3,p {
    padding: 20px 40px 25px 40px;
}

button {
    max-width: 50px;
    align-self: flex-end;
}
a {
    color:blue;
    cursor: pointer;
    font-weight:bold;
}