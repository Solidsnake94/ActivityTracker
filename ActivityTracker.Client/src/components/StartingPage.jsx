import React from "react";

import "../styles/StartingPage.css";

import { withStyles } from "material-ui/styles";

import AppBar from "material-ui/AppBar";
import Toolbar from "material-ui/Toolbar";
import Typography from "material-ui/Typography";
import Paper from "material-ui/Paper";
import Button from "material-ui/Button"

import NavTop from "./DashboardNavTop";

const styles = theme => ({
  button: {
    maxWidth: "50px",
    alignSelf: "flex-end"
  }
});

const StartingPage = props => {
  const login = e => {
    props.history.push("/login");
  };
  const register = e => {
    props.history.push("/register");
  };


  return (
    <div>
      <NavTop />
      <Paper>
        <Typography type="headline" component="h3">
          Welcome activity enthusiast
                </Typography>

        <Typography component="p">
          Create activities and goals and let us track them for you
        </Typography>
        <Typography component="p">
          If it is your first time using our platform you can register  <a onClick={register}>here </a> or you if are are already a member of our community, login in <a onClick={login}>here</a>
        </Typography>
        

      </Paper>
    </div>
  );
};

export default withStyles(styles)(StartingPage);
