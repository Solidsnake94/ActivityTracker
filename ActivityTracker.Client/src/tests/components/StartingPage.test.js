import React from 'react';
import ReactShallowRender from 'react-test-renderer/shallow';

import StartingPage from '../../components/StartingPage';

test('should render startingpage correctly', () => {
    const renderer = new ReactShallowRender();

    renderer.render(< StartingPage />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});