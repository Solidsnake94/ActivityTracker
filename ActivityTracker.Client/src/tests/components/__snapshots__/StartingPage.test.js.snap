// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render startingpage correctly 1`] = `
<StartingPage
  classes={
    Object {
      "button": "StartingPage-button-1",
    }
  }
/>
`;
